JS/Lesson15/gulcin/script.js

```diff
@@ -97,6 +99,8 @@ console.log('------------------------');
 // based on two indices and print it to the console.
 const stringExercise14 =
 	'Databases are an essential component of many applications.';
+console.log('Exercise14: ', stringExercise14.substring(0, 9));
```

Comment:
This code works in a condition where you only have once string that never changes. However, in real life it's more common to have a pattern that occurs in different strings. To practice a situation closer to the real world problem, try to find the indexes (0 and 9) by using indexOf.
Hint: for the end postion, it can be hard to rely on a particular combination of characters but you always know that words "databases" has a particular length, so you could rely on it and the start position instead :)